packages/manager/modules/pci/src/projects/project/inactive/inactive.controller.js

```diff
@@ -0,0 +1,28 @@
+import {
+  CLOUD_PROJECT_STATE,
+  CLOUD_PROJECT_BILLING_STATE,
+} from '../../../constants';
+
+export default class ProjectInactiveCtrl {
+  /* @ngInject */
+  constructor($translate) {
+    this.$translate = $translate;
+  }
+
+  $onInit() {
+    this.projectData = {
+      isSuspended: this.projectStatus === CLOUD_PROJECT_STATE.suspended,
```

Comment:
Why not pass project instance and use project.isSuspended() ?

Reply:
This is done. Thanks :)